packages/liveblocks-core/src/__tests__/room.test.ts

```diff
@@ -156,7 +156,7 @@ describe("room / auth", () => {
       );
 
       room.connect();
-      await waitUntilStatus(room, "authenticating");
+      await waitUntilStatus(room, "connecting");
```

Comment:
Note that "authenticating" and "connecting" are no longer considered two different states. In the new API, there will only be "connecting" as to the user it's uninteresting (and unactionable) to know if the machine is internally authenticating or connecting. It's all part of an attempt to establish a connection.